test/Confluent.Kafka.IntegrationTests/Tests/Consumer_OffsetsForTimes.cs

```diff
@@ -56,7 +54,7 @@ public static void Consumer_OffsetsForTimes(string bootstrapServers, string sing
                 // If empty request, expect empty result.
                 var result = consumer.OffsetsForTimes(new TopicPartitionTimestamp[0], timeout).ToList();
                 Assert.Empty(result);
-
+                
```

Comment:
This wont work

Reply:
yes it will, i explicitly followed the java api here:

https://github.com/confluentinc/confluent-kafka-dotnet/blob/master/src/Confluent.Kafka/Impl/SafeKafkaHandle.cs#L519

Reply:
it is arguable which is better - on the one hand, returning an empty list for this case means code that can take an arbitrarily long list of partitions doesn't need to handle a special case. on the other hand, and empty list may indicate an error. however, other code is going to pick up on that in practice. my view is the former is better, and that java works this way enforces that viewpoint.

Reply:
(but my opinion isn't strong).